Pages/product.css

```diff
@@ -1,50 +1,68 @@
-.page-container{
+.page-container {
     display: flex;
 }
-.filters {
+
+.filter {
     width: 23%;
-    height: 100%;
+    height: 600px;
     box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
-    padding: var(--container-padding);
+    padding: var(--ontainer-padding);
+    overflow-y: scroll;
 }
 
 .filter-head {
```

Comment:
what's filter-head here? Try to give a more meaningful name. Instead of filter-head you can either give filter-div or filter-container.